saleor/graphql/product/filters.py

```diff
@@ -55,6 +55,8 @@ def filter_products_by_collections(qs, collections):
 
 
 def sort_qs(qs, sort_by):
```

Comment:
Could we add a type hint in the signature? `sort_by` sounds like a string to me.